Code/Framework/AzCore/Tests/AZStd/String.cpp

```diff
@@ -980,6 +980,13 @@ namespace UnitTest
         AZ_TEST_ASSERT(*vecIt++ == "Xiph Xlater 10000");
     }
 
+    TEST_F(String, MaxSizeReturnsMoreThan512)
+    {
+        AZStd::wstring foo;
+        printf("Max size of AZStd::wstring is %zu\n", foo.max_size());
```

Comment:
`printf` left behind? When EXPECT_GT fails it'll print out the values.

Reply:
I think this test is meant to be removed

Reply:
This was to validate that max_size was working correctly for `AZStd::string`

Reply:
Will remove